drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/commons/HasPatternPage.java

```diff
@@ -30,5 +30,5 @@
 
     String getEntryPointName();
 
-    List<PatternWrapper> getPatterns();
+    Set<PatternWrapper> getPatterns();
```

Comment:
Changed from a ```List``` to ```Set``` (as the old code contained duplicate entries for multiple actions updating the same pattern)... this has a knock-on effect across a handful of other classes.